addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxThingHandler.java

```diff
@@ -320,6 +350,57 @@ public void updateNetInfo(LanHostsConfig config) {
         }
     }
 
+    public void updateAirPlayDevice(List<AirMediaReceiver> receivers) {
+        if (getThing().getThingTypeUID().equals(FreeboxBindingConstants.FREEBOX_THING_TYPE_AIRPLAY)) {
+            boolean found = false;
+            boolean usable = false;
+            if (receivers != null) {
+                for (AirMediaReceiver receiver : receivers) {
+                    if ((receiver.getName() != null) && receiver.getName().equals(airPlayName)) {
+                        found = true;
+                        usable = Boolean.TRUE.equals(receiver.isVideoCapable());
+                        break;
+                    }
+                }
+            }
+            if (!found) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "AirPlay device not found");
+            } else if (!usable) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
```

Comment:
I am not a specialist of AirPlay but this protocol can be used to render video, audio and photo media. The Freebox API has only implemented video and photo until now. The workaround is to use the video protocol to render audio too. I should comment that in the code.